protocol/contracts/governance/ElderElection.sol

```diff
@@ -93,24 +102,32 @@ contract ElderElection is Ownable, EIP712 {
               nominationFeeToken.safeTransferFrom(account, address(this), nominationFee);
           }
           candidates[discordId] = true;
+          numCandidates += 1;
           emit UpdateNomination(discordId, candidates[discordId]);
```

Comment:
I would personally just hardcode to `true` rather than having another storage lookup
`emit UpdateNomination(discordId, true);`

Reply:
It's a sad programming language that do that optimisation for itself....